src/main/java/com/commercetools/sync/customers/helpers/CustomerSyncStatistics.java

```diff
@@ -0,0 +1,27 @@
+package com.commercetools.sync.customers.helpers;
+
+import com.commercetools.sync.commons.helpers.BaseSyncStatistics;
+
+import static java.lang.String.format;
+
+public class CustomerSyncStatistics extends BaseSyncStatistics {
+
+    public CustomerSyncStatistics() {
```

Comment:
IMO, this constructor is not necessary as it's the same with java's implicit constructor.

Reply:
a1a146907d929e9639f580ebe47f894e7cdce157